plugins/che-dummy-plugin/0.0.1/meta.yaml

```diff
@@ -6,3 +6,12 @@ title: Che Samples Hello World Plugin
 description: A hello world theia plug-in wrapped into a Che Plug-in
 icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
 url: https://github.com/ws-skeleton/che-dummy-plugin/releases/download/untagged-8f3e198285a2f3b6b2db/che-dummy-plugin.tar.gz
+publisher: Red Hat, Inc.
+category: Other
+repository: https://github.com/ws-skeleton/che-dummy-plugin/
+firstPublicationDate: Jan 11, 2019
+latestUpdateDate: Jan 11, 2019
+preview:
```

Comment:
if fields are empty, why are they mandatory ?

Reply:
they're not, and I shall remove them